app/models/invoice.rb

```diff
@@ -29,9 +28,8 @@ def total_revenue
   end
 
   def total_revenue_for_merchant(merchant_id)
-    invoice_items
-    .joins(:item)
-    .where(items: {merchant_id: merchant_id})
+    items
```

Comment:
Interesting, so you have access to items in the invoice model by virtue of the joins table?

Reply:
Yep! Super cool